app/javascript/components/pages/TaskIndex.js

```diff
@@ -1,10 +1,38 @@
 import React, { Component } from 'react';
+import { Row, Col, CardTitle, Card, Container, Button } from 'reactstrap'
+import { NavLink } from 'react-router-dom'
 
 class TaskIndex extends Component {
+
     render() {
         return (
             <>
-                <h1>TaskIndex</h1>
+                <Container>
+                    <h1>Task Index</h1>
+                    <br />
+                    <a href="/task_new" >
+                        <Button color="primary">
+                            Create New Task
+                        </Button>
+                    </a>
+                    <br />
+
+                    {this.props.tasks && this.props.tasks.map(task => {
```

Comment:
You can use optional chaining operator to handle the null checking here, like this: `this.props.tasks?.map`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining